generic/kubernetes/operator-based/keycloak/camunda-keycloak-no-domain-values.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Camunda Platform 8 Values - Keycloak with CNPG Snippet
+# This configuration uses external services managed by Kubernetes operators:
+# - PostgreSQL: CloudNativePG operator with a dedicated cluster for Keycloak
+# - Keycloak: Keycloak operator for identity management (uses 'keycloak-initial-admin' secret)
+#
+# Authentication Secrets:
+# - keycloak-initial-admin: Contains admin username/password for Keycloak (kubernetes.io/basic-auth type, default user: temp-admin)
+# - pg-keycloak-secret: Contains PostgreSQL user credentials for Keycloak
+
+global:
+    # Identity configuration for authentication
+    identity:
+        auth:
+            enabled: true
+            type: KEYCLOAK
+            issuerBackendUrl: http://keycloak-service:8080/auth/realms/camunda-platform
+            # Local testing with port-forwarding
+            # using localhost:8081 -> keycloak-service:8080 (kubectl port-forward svc/keycloak-service 8081:8080)
+            issuer: http://localhost:8081/auth/realms/camunda-platform
+            publicIssuerUrl: http://localhost:8081/auth/realms/camunda-platform
+
```

Comment:
won't work with 8.8

From the road-to-8-8 work:

> The issuerBackendUrl is not supported atm by embedded Identity.
This means that publicIssuerUrl is used and must be accessible by both machines and users.
Only workaround for users is to do atm host file modifications